### ISSUE TITLE
Verbose param for schedulers that don't have it

### ISSUE THREAD

**Issue by sharmalakshay93** [pytorch/pytorch#38726]
## 🚀 Feature
Currently, only the [ReduceLROnPlateau](https://pytorch.org/docs/stable/_modules/torch/optim/lr_scheduler.html#ReduceLROnPlateau) scheduler has the verbose parameter. Proposing that this be added to other schedulers (where applicable; example: [StepLR](https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.StepLR) is an obvious one)

## Motivation
When viewing train logs, it is often not obvious when the LR is being changed. The ReduceLROnPlateau scheduler's verbose parameter allows this to be logged out whenever the LR is changed. 

## Pitch
If the LR is changed, and if the `verbose` is `True`, a message is printed to stdout for each update. [Example](https://github.com/pytorch/pytorch/blob/2de4f245c6b1e1c294a8b2a9d7f916d43380af4b/torch/optim/lr_scheduler.py#L629-L631) from ReduceLROnPlateau: 

```
                if self.verbose:
                    print('Epoch {:5d}: reducing learning rate'
                          ' of group {} to {:.4e}.'.format(epoch, i, new_lr))
```

## Alternatives
n/a

## Additional context
n/a

cc @vincentqb

**Comment by vincentqb** [comment]
Since we already have a verbose parameter in `ReduceLROnPlateau`, I would welcome a PR for this. Do you want to look into this?

**Comment by sharmalakshay93** [comment]
@vincentqb Sure, I can take this up

**Comment by vincentqb** [comment]
Fixed by #41580
